Polyhedron/demo/Polyhedron/Scene_image_item.cpp

```diff
@@ -655,14 +655,14 @@ Scene_image_item::draw(Viewer_interface* viewer) const
 }
 
 template <typename T> const char* whatType(T) { return "unknown"; }    // default
-template <> const char* whatType(float) { return "float"; }    // default
-template <> const char* whatType(double) { return "double"; }    // default
-template <> const char* whatType(char) { return "char"; }    // default
-template <> const char* whatType(boost::uint8_t) { return "uint8 "; }    // default
-template <> const char* whatType(boost::int16_t) { return "uint16"; }    // default
-template <> const char* whatType(boost::uint16_t type) { return "uint16_t"; }    // default
-template <> const char* whatType(boost::int32_t) { return "int32_t"; }    // default
-template <> const char* whatType(boost::uint32_t) { return "uint32_t"; }    // default
+template <> const char* whatType(float) { return "float"; }
+template <> const char* whatType(double) { return "double"; }
+template <> const char* whatType(char) { return "int8_t (char)"; }
+template <> const char* whatType(boost::uint8_t) { return "uint8_t"; }
+template <> const char* whatType(boost::int16_t) { return "uint16_t"; }
```

Comment:
If you want to translate the `uintN_t` and `intN_t`, you can use:
  - `unsigned char` for `uint8_t`,
  - `short` for `int16_t`,
  - `unsigned short` for `uint16_t`.
  - `int` for `int32_t`,
  - `unsigned int` for `uint32_t`.
That list of conversion is true [on a lot of platforms][*], and on all platforms supported by CGAL.

[*]: https://en.wikipedia.org/wiki/64-bit_computing#64-bit_data_models